packages/tooltip/src/use-tooltip.ts

```diff
@@ -60,12 +60,12 @@ export interface UseTooltipProps {
 
 export function useTooltip(props: UseTooltipProps = {}) {
   const {
-    showDelay = 0,
-    hideDelay = 0,
+    showDelay = 200,
+    hideDelay = 200,
```

Comment:
Based on the Typescript Interface defaults,
![image](https://user-images.githubusercontent.com/39694575/86509368-0f5ba680-be05-11ea-86b7-53544a2c49c5.png)

I updated the defaults

Reply:
If we make the above change to `onOpen`/`onClose`, we should probably change these to `openDelay` and `closeDelay` as well.

Reply:
Please update the js doc and default value to `0`, that's what we hard in the previous version. 

I guess I skipped updating it. Thank you

Reply:
Done, I made these changes.